index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Sofia">
+
+    <title>portfolio</title>
+</head>
+
+<body>
+    <header class="header">
+        <div class="bar1"> 9:41
+            <img src="./images/Group 4.png" alt="icons"></div>
+        <div class="bar2">
+            <h1>TRACY MIRANJA</h1>
```

Comment:
- [OPTIONAL] Kindly wrap your logo in an `<a>` tag. This is important because you will be using it to navigate to the home page